SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/Constants.cs

```diff
@@ -32,5 +35,17 @@ public static string AppCenterFilesDirectoryPath
         public static string AppCenterDatabasePath = Path.Combine(AppCenterFilesDirectoryPath, "Logs.db");
 
         public static string LocalAppData => Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+
+        public static string UserName
```

Comment:
Why do we need this?

Reply:
It need for replacing username value in tests.